style.css

```diff
@@ -529,3 +529,212 @@ p {
 a {
   text-decoration: none;
 }
+
+.footer {
+  background: #e5e5e5;
+}
+
+.containers {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 90px;
+  position: absolute;
+  width: 360px;
+  height: 844px;
+  left: 0;
+  top: 6038px;
+  background: #e5e5e5;
+}
+
+.textheads {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 320px;
+  height: 216px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.contactheads {
+  width: 320px;
+  height: 220px;
+  font-family: 'Inter', sans-serif;
```

Comment:
- [ ] Kindly make sure you are referring to the design provided in Figma:
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193680579-e73ed883-f516-44d6-bc66-9a4b437f07c9.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193680721-461845bb-2601-4449-baf1-d8a63020d7b3.png)